tests/cupy_tests/core_tests/test_internal.py

```diff
@@ -70,34 +70,34 @@ def test_different_size(self):
 class TestGetCContiguity(unittest.TestCase):
 
     def test_zero_in_shape(self):
-        assert internal.get_c_contiguity((1, 0, 1), (1, 1, 1), 3)
+        assert internal.get_c_contiguity((1, 0, 1), (1, 1, 1), 3) is True
```

Comment:
`get_c_contiguity` is an internal interface, so we don't have to check the return value is of `bool` type.